src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -1878,7 +1878,7 @@ public function testIfNotifierTransportsAreKnownByFrameworkExtension()
     {
         $container = $this->createContainerFromFile('notifier');
 
-        foreach ((new Finder())->in(\dirname(__DIR__, 4).'/Component/Notifier/Bridge')->directories()->depth(0)->exclude('Mercure') as $bridgeDirectory) {
+        foreach ((new Finder())->in(\dirname(__DIR__, 4).'/Component/Notifier/Bridge')->directories()->depth(0)->exclude(['Mercure', 'Iterable']) as $bridgeDirectory) {
```

Comment:
```suggestion
        foreach ((new Finder())->in(\dirname(__DIR__, 4).'/Component/Notifier/Bridge')->directories()->depth(0)->exclude(['Mercure']) as $bridgeDirectory) {
```

Please remove and let the test fail for now, thanks